src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -2221,5 +2225,22 @@ void GraphOptimizer::reshapeRnnSeq(Graph &graph) {
     }
 }
 
```

Comment:
I propose to add a corresponding description to the transformation regarding the motivation.
i.e.
"Removes redundant Convert node.
Example: BF16 model output is forced by post-processing API which results in the following subgraph:
Node [FP32] -> Convert [BF16] -> Outputs [BF16]
After EnforceBF16 routine the subgraph becomes:
Node [BF16] -> Convert [BF16] -> Outputs [BF16]
So Convert is redundant."